app/controllers/katello/api/v2/simple_content_access_controller.rb

```diff
@@ -11,7 +10,7 @@ class Api::V2::SimpleContentAccessController < Api::V2::ApiController
     api :GET, "/organizations/:organization_id/simple_content_access/eligible",
       N_("Check if the specified organization is eligible for Simple Content Access")
     def eligible
-      eligible = @organization.upstream_consumer.simple_content_access_eligible?
+      eligible = @organization.simple_content_access_eligible?
```

Comment:
We use this here https://github.com/Katello/hammer-cli-katello/blob/master/lib/hammer_cli_katello/simple_content_access.rb#L5

Since SCA is the default now and no longer tied to upstream, do you think it's worth it to add a deprecation warning, then in 4.9 we can look at pulling it out.

Reply:
yeah, we should probably do that and remove both the Katello and Hammer parts in 4.9.